less/datepicker.less

```diff
@@ -5,7 +5,7 @@
 		width: 220px;
 	}
 	direction: ltr;
-	&&-rtl {
+	&-rtl {
```

Comment:
Isn't this a somewhat soft BC break as users could use the `.datepicker.datepicker-rtl` selector. I also think this a bug but by fixing it we break BC. What do you think? \cc @Azaret @hebbet